pkg/fftypes/transaction.go

```diff
@@ -19,8 +19,10 @@ package fftypes
 type TransactionType = FFEnum
 
 var (
-	// TransactionTypeNone indicates no transaction should be used for this message/batch
+	// TransactionTypeNone deprecreated - replaced by TransactionTypeUnpinned
 	TransactionTypeNone TransactionType = ffEnum("txtype", "none")
+	// TransactionTypeUnpinned indicates no transaction should be used for this message/batch
```

Comment:
I think the wording should indicate that the message will be sent without pinning any evidence to the blockchain.

There _is_ a FireFly transaction (although there's no blockchain transaction) - so want to avoid confusion there.